tests/framework/web/AssetBundleTest.php

```diff
@@ -27,6 +28,24 @@ protected function setUp()
         Yii::setAlias('@testAssetsPath', '@webroot/assets');
         Yii::setAlias('@testAssetsUrl', '@web/assets');
         Yii::setAlias('@testSourcePath', '@webroot/assetSources');
+
+        // clean up assets directory
+        $handle = opendir($dir = Yii::getAlias('@testAssetsPath'));
+        if ($handle === false) {
+            throw new \Exception("Unable to open directory: $dir");
```

Comment:
not sure, what do you mean? this is just to fail tests in case of permission issues.